index.html

```diff
@@ -24,125 +41,629 @@
-              Items
-            </button>
+            <a
+              data-nav
+              href="https://html.spec.whatwg.org/multipage/popover.html#dom-popover"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Spec</a
+            >
+          </li>
+          <li>
+            <a
+              data-nav
+              href="https://developer.mozilla.org/en-US/docs/Web/API/Popover_API"
+              target="_blank"
+              rel="noopener noreferrer"
+              >API</a
+            >
+          </li>
+          <li>
+            <a
+              data-nav
+              href="https://github.com/oddbird/popover-polyfill"
+              target="_blank"
+              rel="noopener noreferrer"
+              >Source</a
+            >
           </li>
         </ul>
-        <div id="popover11-1" popover="auto" style="left: 200px">
-          <ul>
-            <li><button>Nested</button></li>
-            <li><button>Menu</button></li>
-            <li><button>Items</button></li>
+      </nav>
+    </header>
+    <main>
+      <header class="main-header">
+        <h1 id="title">
+          <span class="subtitle">Polyfill Demo</span>
+          <span data-popover-title>Popover</span>
+          <span data-popover-title="attribute">Attribute</span>
+        </h1>
+        <div class="description">
+          <p>
+            OddBird’s Popover Attribute Polyfill – built in collaboration with
+            <a href="https://github.com/keithamus">Keith Cirkel</a> and used in
+            production by GitHub – lets developers preview the upcoming
+            mechanism for
+            <a href="https://open-ui.org/components/popover.research.explainer/"
+              >displaying popover content</a
+            >
+            on top of other page content, drawing the user’s attention to
+            specific important information or actions that need to be taken.
+          </p>
+          <p>
+            This polyfills the HTML <code>popover</code> attribute and
+            <code>showPopover</code>, <code>hidePopover</code>, and
+            <code>togglePopover</code> methods on <code>HTMLElement</code>, as
+            well as the <code>popovertarget</code> and
+            <code>popovertargetaction</code> attributes on
+            <code>&lt;button&gt;</code> elements.
+          </p>
+        </div>
+      </header>
+      <div id="popovers">
+        <div id="defaultPopover" data-popover popover>Default Popover</div>
+        <div
+          id="manualPopover"
+          data-popover
+          popover="manual"
+          style="bottom: 150px"
+        >
+          Manual Popover
+        </div>
+        <div id="emptyStatePopover" data-popover popover="">
+          Empty State Popover
+        </div>
+        <div
+          id="showHidePopover"
+          data-popover
+          popover="manual"
+          style="top: 150px"
```

Comment:
Because `manual` popovers sometimes stay on the screen (they aren't dismissed when another popover opens), it was annoying both in testing and playing with this demo to have them overlap other popovers. I added this offset to the 2 manual popovers, but maybe there's a better suggestion?

Reply:
Makes sense. I kept the offset size but just moved them to the css file in https://github.com/oddbird/popover-polyfill/pull/187/commits/f699fb1ba4db45a30f9de20f080bcd3dc2720d98